### PR TITLE
adding files to bundle template, update massdriver.yaml

### PR DESCRIPTION
**Pull request by WillBeebe**
### Added
- pkg/bundle/templates/terraform/Makefile
- pkg/bundle/templates/terraform/tf_helper.rb

### Bugfix
- bundle template was missing steps in `massdriver.yaml`

### CODE REVIEW

File: pkg/bundle/templates/terraform/massdriver.yaml [hunk 1]
```diff
@@ -6,6 +6,10 @@ ref: github.com/YOUR_NAME_HERE/{{ .Name }}
 access: "{{ .Access }}"
 type: "{{ .Type }}"
 
+steps:
```
Comment (coryodaniel) [inline pkg/bundle/templates/terraform/massdriver.yaml:9]: Lets check in to see if @chrisghill got the fix into xo for this. I'd like not to expose this functionality to people. Its something for them to mess up with no feature to gain.

File: pkg/bundle/templates/terraform/tf_helper.rb [hunk 2]
```diff
@@ -0,0 +1,58 @@
+#! ruby
```
Comment (coryodaniel) [inline pkg/bundle/templates/terraform/tf_helper.rb:1]: I think this is fine to add but it'll be getting replaced fairly soon. I was planning to convert this to go using terratest so we could start automating the tf tests we are doing locally in CI.

**Review by coryodaniel — APPROVED** [review]
Check w/ chris re: steps. I know it makes all of our bundles broken so we want to add it, but fixing it in xo is the better choice if it can happen quickly.

See notes re ruby script.

File: pkg/bundle/templates/terraform/massdriver.yaml (hunk 1 above)
Comment (WillBeebe) [reply pkg/bundle/templates/terraform/massdriver.yaml:9]: I'll yank it from the template

**Pull request merged**